src/components/BooksForm.js

```diff
@@ -1,11 +1,29 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { setBook } from '../redux/books/books';
 
 export default function BookForm() {
+  const dispatch = useDispatch();
+  function submitHandler(e) {
+    e.preventDefault();
+    dispatch(setBook(
+      {
+        gender: e.target.children[1].value,
+        author: 'New Author',
```

Comment:
- [ ] According to requirements, please create a local state with `title` and `author` and they should have their inputs that update the local state every time they update (`onChange`)

Reply:
Its need to be in every component? As the local state is handle in the Books.js with setState in an array of objects, the BooksForm is passing the data to redux for make it global and stored in the App trough Books.js

Reply:
@codepantha as Booksform the only function is get data and not display it thats way is only passing data to Redux, and Books.js handle the storage of the data.

Reply:
@dgonzalesi  What you did works quite alright but the requirement was to make sure there was a local state